code/src/index.css

```diff
@@ -5,9 +5,226 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color:#ccb79f;
+  @import url('https://fonts.googleapis.com/css2?family=Inconsolata&display=swap');
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+main {
+  max-width: 500px;
+  margin: 0 auto;
+  min-height: 700px;
+}
+
+.header {
+  text-align: left;
+  background-image: url(./assets/todo.jpg);
+  background-size: cover;
+  background-position: center;
+  height: 250px;
+}
+
+.header h1 {
+  font-family: 'Inconsolata', monospace;
+  margin:0;
+  margin-left:10px;
```

Comment:
Set your prettier to auto-format your html/js/css files to avoid small issues like lack of space after `:`